frontend/src/scenes/error-tracking/errorTrackingDataNodeLogic.tsx

```diff
@@ -22,48 +22,48 @@ export const errorTrackingDataNodeLogic = kea<errorTrackingDataNodeLogicType>([
     })),
 
     actions({
-        mergeGroups: (indexes: number[]) => ({ indexes }),
-        assignGroup: (recordIndex: number, assigneeId: number | null) => ({
+        mergeIssues: (indexes: number[]) => ({ indexes }),
+        assignIssue: (recordIndex: number, assigneeId: number | null) => ({
             recordIndex,
             assigneeId,
         }),
     }),
 
     listeners(({ values, actions }) => ({
-        mergeGroups: async ({ indexes }) => {
-            const results = values.response?.results as ErrorTrackingGroup[]
+        mergeIssues: async ({ indexes }) => {
+            const results = values.response?.results as ErrorTrackingIssue[]
 
-            const groups = results.filter((_, id) => indexes.includes(id))
-            const primaryGroup = groups.shift()
+            const issues = results.filter((_, id) => indexes.includes(id))
+            const primaryIssue = issues.shift()
 
-            if (primaryGroup && groups.length > 0) {
-                const mergingFingerprints = groups.map((g) => g.fingerprint)
-                const mergedGroup = mergeGroups(primaryGroup, groups)
+            if (primaryIssue && issues.length > 0) {
+                const mergingIds = issues.map((g) => g.id)
+                const mergedIssue = mergeIssues(primaryIssue, issues)
 
                 // optimistically update local results
```

Comment:
Is there some way we need to go back and patch up this optimistic update once the API request has finished? Does this get handled automatically through onLoad values or something?

Reply:
Yeah, we don't handle the error case very well. Added a comment to come back to it. Worst case is that the UI will be fixed after a page refresh